tools/dev-release.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/sh -xe
+# Release dev packages to PyPI
+
+version="0.0.0.dev$(date +%Y%m%d)"
+DEV_RELEASE_BRANCH="dev-release"
+# TODO: create a real release key instead of using Kuba's personal one
+RELEASE_GPG_KEY="${RELEASE_GPG_KEY:-148C30F6F7E429337A72D992B00B9CC82D7ADF2C}"
+
+# port for a local Python Package Index (used in testing)
+PORT=${PORT:-1234}
+
+# subpackages to be released
+SUBPKGS=${SUBPKGS:-"acme letsencrypt_apache letsencrypt_nginx letshelp_letsencrypt"}
+subpkgs_dirs="$(echo $SUBPKGS | sed s/_/-/g)"
+# letsencrypt_compatibility_test is not packaged because:
+# - it is not meant to be used by anyone else than Let's Encrypt devs
+# - it causes problems when running nosetests - the latter tries to
+#   run everything that matches test*, while there are no unittests
+#   there
+
+tag="v$version"
+mv "dist.$version" "dist.$version.$(date +%s).bak" || true
+git tag --delete "$tag" || true
```

Comment:
There is no such thing as "tag reuse" if you're using signed tags. You need to remove the old one first, before recreating it.

Note that everything happens on a clone of you local git copy, so there is no chance you accidentally remove the tag from the GitHub, if that's what you're afraid of.

To be honest, I've rewritten the script a couple of times and this is a remnant of the version, where I didn't do `git clone` as the first step.